src/test/groovy/io/openliberty/tools/gradle/InstallLiberty_installDir_missing_wlp_Test.groovy

```diff
@@ -45,4 +45,16 @@ class InstallLiberty_installDir_missing_wlp_Test extends AbstractIntegrationTest
         String output = result.getOutput()
         assert output.contains("path does not reference a wlp folder") : "Expected warning about installDir path not containing wlp"
     }
+
+    @Test
+    void test_installLiberty_installDir__cli_property() {
+        BuildResult result = GradleRunner.create()
+            .withProjectDir(buildDir)
+            .forwardOutput()
+            .withArguments('installLiberty', '-Pliberty.installDir=installDir-valid-install/build/wlp', '-i', '-s')
```

Comment:
After adding the logic to check the path above, you can add another test here to pass in a path with `-Pliberty.installDir` that does not end with `/wlp`.